lib/lndclient/LndClient.ts

```diff
@@ -72,6 +72,9 @@ class LndClient extends SwapClient {
     LTC: 2.5,
   };
 
+  // This's the hash of "This's a test invoice".
+  private static RHASH_INVOICE_CHECK = '39990D19D34FC9C27284BC7739CF5794E521CDDA3B1E0580F9DD6987FD2210DB';
```

Comment:
This value has to change before every attempt. Else it will fail every time after the first successful check:

![image](https://user-images.githubusercontent.com/8138838/94037246-73b75200-fdc5-11ea-9ba7-cfe9cc60477c.png)

`removeInvoice` does *not* actually remove the invoice but just cancel the XUD subscription streams and tell LND that the invoice will not be settled in the future. It is more like abandoning the invoice than removing it.